### PR TITLE
Fix horizontal scroll. Fixes #43

### LINKED ISSUE #43: Horizontal scroll
There's horizontal scroll on my browser
horizontalscroll

### PR DESCRIPTION
**Pull request by wailingtam**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -45,22 +45,22 @@ <h4 id="lenny-face">(๑˃ᴗ˂)ﻭ</h4>
...
-					<a class="fa fa-2x fa-youtube" target="_blank" href="https://www.youtube.com/channel/UCiiRorGg59Xd5Sjj9bjIt-g"></a>
-					<a class="fa fa-2x fa-facebook-official" target="_blank" href="https://facebook.com/hackupc"></a>
-					<a class="fa fa-2x fa-github" target="_blank" href="https://github.com/hackupc"></a>
-					<a class="fa fa-2x fa-medium" target="_blank" href="https://medium.com/@hackupc"></a>
-				</div>
-			</div>
+	</div><!-- wrapper end -->
 
-			<a href="https://w2017.hackupc.com" target="_blank">Winter 2017 site</a>
+	<div class="footer">
```
Comment (casassg) [inline src/index.html]: how is this a git diff?

Comment (dasilvacontin) [inline src/index.html]: Indentation has changed mr @casassg.

Comment (dasilvacontin) [inline src/index.html]: No longer inside wrapper element.

**Comment by dasilvacontin** [comment]
The changeset fixes the issue. Ty @wailingtam! ❤️

File: src/index.html (hunk 1 above)
Comment (casassg) [inline src/index.html]: I didnt realize, thought it was inside the wrapper still

**Comment by dasilvacontin** [comment]
I agree the diff visually sucks.

**Pull request merged**